lib/client.ml

```diff
@@ -101,16 +109,16 @@ let list channel =
   channel.read buf
   >>= fun () ->
   match Announcement.unmarshal buf with
-  | `Error e -> fail e
-  | `Ok kind ->
+  | Result.Error e -> fail e
```

Comment:
it is better to `open Result` at the beginning of the file (and this is not even needed for OCaml>4.03)